crates/api_models/src/enums.rs

```diff
@@ -461,9 +461,9 @@ pub enum FieldType {
     UserAddressState,
     UserAddressCountry,
     UserBlikCode,
-    FieldsComplete,
     UserBillingName,
     UserBank,
+    UserCurrency { options: Vec<String> },
```

Comment:
Can we keep this as `&'static str` instead? We can take this up on a separate PR.

Reply:
yes, there will be another pr in which I ll add support for other connectors. Will add this change in that pr